src/api/hooks.ts

```diff
@@ -168,30 +168,36 @@ export function useRasterAlarmTriggers() {
   return triggers;
 }
 
-interface WrappedConfig {
+// We use different client config objects (for the configuration of the whole app, and for
```

Comment:
We now use two different client config objects. The messages one is updated by the client code, I think the first time we use that.